### PR TITLE
feat: add component card and section of courses

### PR DESCRIPTION
**Pull request by waelelsafty07**

### CODE REVIEW

File: src/components/CardDetails/CardDetails.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+const theme = createTheme({
+  palette: {
+    starRating: {
+      gold: "#ff9800",
+      gray: "#6a6f73",
+    },
+  },
+});
+const CardDetails = ({ image, title, author, rating, price, people }) => {
```
Comment (shereen-fathy) [inline src/components/CardDetails/CardDetails.js]: `CouseCard or CourseDetails` will be a more appropriate name here

File: src/components/Courses/Courses.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      );
+    })
+  ) : (
+    <h1>Not Found</h1>
+  );
+
+  return (
+    <>
+      <Container sx={{ border: 1, p: 5 }}>
+        <h2>Expand your career opportunities with Python</h2>
```
Comment (shereen-fathy) [inline src/components/Courses/Courses.js]: according to the requirements, you should take courses, title, and description as a prop to this component, right?

**Review by shereen-fathy — CHANGES_REQUESTED** [review]
small changes just to match task requirements

File: src/components/Courses/Courses.js (hunk 2 above)
Comment (waelelsafty07) [reply src/components/Courses/Courses.js]: I, didn't know about that, i will fix that

Comment (waelelsafty07) [reply src/components/Courses/Courses.js]: Sorry for forgetting that

**Review by shereen-fathy — APPROVED** [review]
Thank you for the changes

**Pull request merged**